### PR TITLE
CIV-7498 SDO Latest Update for LIP Defendant

### PR DESCRIPTION
**Pull request by LeonardoPalmeiro**
### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-7161

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by laperla11 — APPROVED** [review]

**Review by nagaraju-deekonda — APPROVED** [review]

**Review by TatsyMaslova — DISMISSED** [review]

**Review by neeta-hmcts — DISMISSED** [review]

**Review by kenneth-hmcts — DISMISSED** [review]

File: charts/civil-citizen-ui/values.preview.template.yaml [hunk 1]
```diff
@@ -26,7 +26,7 @@ civil-service:
   java:
     applicationPort: 4000
     releaseNameOverride: ${SERVICE_NAME}-civil-service
-    image: 'hmctspublic.azurecr.io/civil/service:latest'
+    image: 'hmctspublic.azurecr.io/civil/service:pr-2478'
```
Comment (neeta-hmcts) [inline charts/civil-citizen-ui/values.preview.template.yaml:29]: revert this change

Comment (LeonardoPalmeiro) [reply charts/civil-citizen-ui/values.preview.template.yaml:29]: @rajakm if I revert this I'm not able to create any claim with SDO. shall I revert or not?

**Review by neeta-hmcts — DISMISSED** [review]

File: charts/civil-citizen-ui/values.preview.template.yaml (hunk 1 above)
Comment (PaulH-HMCTS) [reply charts/civil-citizen-ui/values.preview.template.yaml:29]: Yeah revert it from here before merging into master - we will have a separate Test PR for the QA part (which won't be merged)

**Review by PaulH-HMCTS — APPROVED** [review]

**Review by kenneth-hmcts — APPROVED** [review]

**Review by laperla11 — APPROVED** [review]

**Review by PaulH-HMCTS — DISMISSED** [review]
